internal/proto/controller/storage/kms/store/v1/data_key_version_destruction_job_progress.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package controller.storage.kms.store.v1;
+
+import "controller/storage/timestamp/v1/timestamp.proto";
+
+option go_package = "github.com/hashicorp/boundary/internal/kms/store;store";
+
+message DataKeyVersionDestructionJobProgress {
```

Comment:
Is there a PK (I don't see a gorm tag for it)?

Reply:
This is a view, it doesn't have a primary key.